woocommerce/woocommerce-gateway-stripe
# feat: express payment methods with new settings

# Changes proposed in this Pull Request:

Same thing we did here: https://github.com/Automattic/woocommerce-payments/pull/2036

The new settings don't have the same options for the payment methods. Which means that the frontend should also adapt.
Options are migrated here: https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1883

# Testing instructions
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&area=payment_requests
- Change some settings
- Save
- Ensure you're _not_ using localhost (use Jurassic Tube or ngrok instead)
- Ensure you have some cards saved on GPay or on Chrome autofill or on ApplePay - see requirements here: https://stripe.com/docs/js/payment_request/can_make_payment
- Settings are honored on the frontend

PLEASE NOTE: I noticed that changing the settings often might not reflect the changes immediately to the frontend, probably due to some caching. If the changes aren't reflected, just wait a little bit. Example: https://d.pr/i/t8b93E
